src/utils/extension/browserApi.js

```diff
@@ -0,0 +1,6 @@
+/*global chrome*/
+/*global browser*/
+
+import { runningInChromeDesktop } from "../misc/browserDetection";
+
+export const BROWSER_API = runningInChromeDesktop() ? chrome : browser;
```

Comment:
@tfnribeiro - why does this appear in this branch?

Reply:
So at some point I had experimented with adding the BrowserAPI to see if that would fix our issues to communicate with the extension. However, when that didn't work - I reverted that change:

![image](https://github.com/zeeguu/web/assets/17390076/ee3baae4-861d-4988-b423-a758f2b1155c)

Not sure why this wasn't remove for this branch?